src/conn/conn_ckpt.c

```diff
@@ -38,6 +38,13 @@ __ckpt_server_config(WT_SESSION_IMPL *session, const char **cfg, bool *startp)
 	if (conn->ckpt_usecs != 0 ||
 	    (conn->ckpt_logsize != 0 &&
 	    FLD_ISSET(conn->log_flags, WT_CONN_LOG_ENABLED))) {
+		/*
+		 * If non-zero, use a minimum of the log file size.  The logging
+		 * subsystem has already been initialized.
+		 */
+		if (conn->ckpt_logsize != 0)
+			conn->ckpt_logsize = WT_MAX(
+			    conn->ckpt_logsize, conn->log_file_max);
```

Comment:
I've got two questions about this change:
* Can we guarantee that log_file_max is always configured before getting here.
* Does this work as expected when reconfiguring?

I'm more inclined to put a hard lower bound on the volume of log records, rather than tie it to another configuration option. That makes documentation simpler, and leads to less configuration corner cases.